src/occa/internal/bin/occa.hpp

```diff
@@ -6,6 +6,7 @@
 namespace occa {
   namespace bin {
     cli::command buildOccaCommand();
+    bool runTranslate(const json &args);
```

Comment:
Had to expose this function so I don't have to redundantly implement it again in tests. Hope that is okay.